File: samples/demo-projects-js/project-b/mathHelper.js
class MathHelper {
  constructor(amount) {
    this.amount = amount;
  }

  plus(operand) {
    const outcome = this.amount + operand;
    return outcome;
  }

  minus(operand) {
    const outcome = this.amount - operand;
    return outcome;
  }

  times(multiplier) {
    let accumulator = 0;
    for (let index = 0; index < multiplier; index = index + 1) {
      accumulator = accumulator + this.amount;
    }
    return accumulator;
  }

  greaterThanZero() {
    if (this.amount > 0) {
      return true;
    } else {
      return false;
    }
  }
}

File: samples/demo-projects-python/project-b/math_helper.py
class MathHelper:
    def __init__(self, amount):
        self.amount = amount

    def plus(self, operand):
        outcome = self.amount + operand
        return outcome

    def minus(self, operand):
        outcome = self.amount - operand
        return outcome

    def times(self, multiplier):
        accumulator = 0
        for index in range(multiplier):
            accumulator = accumulator + self.amount
        return accumulator

    def greater_than_zero(self):
        if self.amount > 0:
            return True
        else:
            return False
